app/ide-desktop/lib/client/src/index.ts

```diff
@@ -279,7 +279,8 @@ class App {
                 const useFrame = this.args.groups.window.options.frame.value
                 const macOS = process.platform === 'darwin'
                 const useHiddenInsetTitleBar = !useFrame && macOS
-                const useVibrancy = this.args.groups.window.options.vibrancy.value
+                const useVibrancy = (this.args.groups.window.options.vibrancy.value &&=
```

Comment:
Is the assignment here intentional? If so, I’d split that into separate lines.

Reply:
thanks, fixed